Hangman.js

```diff
@@ -0,0 +1,68 @@
+var randomWord = require("random-word");
+var inquirer = require("inquirer");
+var Word = require("./word.js").Word;
+
+function play() {
+  var random = new Word(randomWord());
+  var tries = 10;
+  var guessed = [];
```

Comment:
The data doesn't leave the function so it could just be written using this instead of var.